autopilot/hostfilter.go

```diff
@@ -102,6 +102,15 @@ func (f *hostFilter) withRemainingFundsFilter(cfg Config) *hostFilter {
 	return f
 }
 
+func (f *hostFilter) withScoreFilter(cfg Config, threshold float64) *hostFilter {
+	if score := totalHostScore(cfg, f.host); score < threshold {
+		f.metadata.GoodForUpload = false
+		f.metadata.GoodForRenew = false
```

Comment:
I think we should reevaluate whether we truly need GFU/GFR, or if they're just holdovers from `siad`. That is: why do we track these things as explicit booleans, rather than computing them on-demand from other values?